netbox_data_import/tests/js/preview_row_controls.test.js

```
@@ -0,0 +1,242 @@
+  "utf8",
+);
+
+let controllerLoaded = false;
+
+/* The head script runs before the table exists, exactly as it does while a large
+ * preview streams to the browser. NetBox exposes Bootstrap's Modal as a global. */
+function loadController() {
+  document.body.innerHTML = "";
+  window.Modal = Modal;
+  if (!controllerLoaded) {
+    window.eval(controllerSource);
```

Biome (2.5.6)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)